examples/complete/variables.tf

```diff
@@ -50,3 +50,15 @@ variable "existing_at_instance_crn" {
   description = "Optionally pass an existing activity tracker instance CRN to use in the example. If not passed, a new instance will be provisioned"
   default     = null
 }
+
+variable "cos_plan" {
+  description = "Plan to be used for creating cloud object storage instance. Only used if 'create_cos_instance' it true."
+  type        = string
+  default     = "cos-one-rate-plan"
+}
+
+variable "bucket_storage_class" {
+  description = "Storage plan to be applied to your cos instance, only valid one rate plan storage class."
+  type        = string
+  default     = "onerate_active"
+}
```

Comment:
This is confusing because the example creates 2 buckets, but you are only using these variables on the second bucket. I would just hard code them in the example instead of using variables. Also ensure the example readme is update to specify the plan deployed for each bucket

Reply:
@jor2 Same thing should be done with `bucket_storage_class` variable. The variable is only being used by one of the buckets in the example

Reply:
Needed to create new example as in complete example cos_bucket2 uses one rate cos_bucket1 as existing instance and enables cross region support, but one rate plan doesn't support cross region.